src/realm/table.hpp

```diff
@@ -503,6 +507,17 @@ class Table {
     void set_null(size_t column_ndx, size_t row_ndx, bool is_default = false);
     void set_null_unique(size_t col_ndx, size_t row_ndx);
 
+    template <class T>
+    void set(size_t c, size_t r, T value, bool is_default = false);
+
+    template <class T>
+    void set_list(size_t c, size_t r, const std::vector<T>& list);
+
+    TableView get_list(size_t column_ndx, size_t row_ndx);
```

Comment:
(I wrote this comment before I saw that you had already made a method returning a std::vector of the payload, which was what I wanted)